cluster-scope/base/core/serviceaccounts/vault-nerc-secret-reader/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: vault-nerc-secret-reader
+  namespace: external-secrets-config
```

Comment:
In OPF we actually don't have a vault-secret-reader sa in the ESO namespace. We do have vault-secret-reader in each namespace which utilizes vault. We will need a token-reviewer in the  external-secrets-config namespace with auth delegtion crb:
```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: eso-tokenreview
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: system:auth-delegator
subjects:
  - kind: ServiceAccount
    name: eso-vault-auth
    namespace: external-secrets-config
```

Reply:
That's not to say we can't do it this way though

Reply:
Yeah agreed I think this makes sense. Also simplifies infra a bit and only admins should have access to this cluster anyway if I remember correctly. 
I also think in the prod cluster per namespace is the way to go.